src/lib/Input.svelte

```diff
@@ -15,7 +15,17 @@
     let innerText = "";
     let messageReceived = true;
 
+    /**
```

Comment:
Beautiful comments @shengh318 ! But your functions are named so well that I think they're redundant -- I personally always prefer well-named functions over comments, as comments are harder to keep up to date (and though good comments are the best, bad/out-of-date comments are worse than no comments since they can confuse people)

Reply:
Oh yeah. So MIT's CS class has like ingrained this doc-string documentation into every single 6-3 student so that's why I was writing the docstrings.